flight/Modules/ManualControl/tablet_control.c

```diff
@@ -207,13 +208,8 @@ static int32_t tabletInfo_to_ned(TabletInfoData *tabletInfo, float *NED)
 	GPSPositionData gpsPosition;
 	GPSPositionGet(&gpsPosition);
 
-	float lat = homeLocation.Latitude / 10.0e6f * DEG2RAD;
-	float alt = homeLocation.Altitude;
-
 	float T[3];
-	T[0] = alt+6.378137E6f;
-	T[1] = cosf(lat)*(alt+6.378137E6f);
-	T[2] = -1.0f;
+	LLA_linearization_float(homeLocation.Latitude, homeLocation.Altitude, T);
```

Comment:
This shouldn't be done every loop. It's not a blocker for this PR, but if you wanted to you could move it out.